docs/src/prototypes/VirtualizedTable/VirtualizedTable.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import { Table } from '@fluentui/react'
+import { AutoSizer, Table as ReactVirtualizedTable } from 'react-virtualized'
+import getItems from './itemsGenerator'
+
+interface TableVirtualizerProps {
+  renderedItems: React.ReactElement[]
+}
+
+function TableVirtualizer(props: TableVirtualizerProps) {
+  const rowGetter = ({ index }) => {
+    return props.renderedItems[index]
+  }
+
+  const rowRenderer = ({ index, style, rowData }) => {
+    return React.cloneElement(props.renderedItems[index], {
+      style,
+    })
+  }
+
+  return (
+    <AutoSizer disableHeight>
+      {({ width }) => (
+        <ReactVirtualizedTable
+          disableHeader={true}
+          height={600}
+          rowCount={props.renderedItems.length}
+          width={width}
+          rowHeight={80}
```

Comment:
Are we sure that all rows are 80? There is also CellMeasurer as part of `react-virtualized` if you don't want to make this assumption :)

Reply:
It's just for a demo, the prototype purpose is to show a way to use react-virtualized with our `Table` component, so I didn't want to overload it with other things. I can add it as a note in the description, but would like to leave it like that, if you agree :)

Reply:
Sure, I was just asking. Feel free to add a comment.